independent-publisher-connectors/OK dokument/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "success": {
+                  "type": "boolean",
+                  "description": "success",
+                  "title": "success",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}
+          }
+        },
+        "summary": "Document Delete",
+        "operationId": "DocumentDelete",
+        "description": "Document Delete",
+        "parameters": [
+          {
+            "name": "Document ID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important",
+            "description": "Document ID",
+            "x-ms-summary": "Document ID"
+          }
+        ]
+      }
+    },
+    "/signatureRequest/{signatureRequestId}": {
+      "x-ms-notification-content": {
+        "description": "Status ok once document is signed",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "code": {
+              "type": "string",
+              "description": "code",
+              "title": "code"
+            },
+            "message": {
+              "type": "string",
+              "description": "message",
+              "title": "message",
+              "x-ms-visibility": "internal"
+            },
+            "result": {
+              "type": "string",
+              "description": "result",
+              "title": "result",
+              "x-ms-visibility": "internal"
+            },
+            "statusCode": {
+              "type": "integer",
+              "format": "int32",
+              "description": "statusCode",
+              "title": "statusCode"
+            },
+            "url": {
+              "type": "string",
+              "description": "url",
+              "title": "url",
+              "x-ms-visibility": "internal"
+            },
+            "success": {
+              "type": "boolean",
+              "description": "success",
+              "title": "success",
+              "enum": [
+                true,
+                false
+              ],
+              "x-ms-visibility": "internal"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@SoliteaSlovensko we can keep empty schemas for trigger operations. For action operations they have to be defined or deleted.